NuSpecs/Microsoft.WinRTC.libwebrtc.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.WinRTC.libwebrtc</id>
+        <version>84.0.3-alpha</version>
```

Comment:
Is there a way to have this auto generated?

Reply:
Yes, the package.cmd file overrides the version.